docs/reference/components/sinks.cue

```diff
@@ -407,8 +407,18 @@ components: sinks: [Name=string]: {
 				if features.send.batch != _|_ {
 					if features.send.batch.enabled {
 						buffers_batches: {
+							_parameter: string
+							if features.send.batch.max_bytes != _|_ && features.send.batch.max_events == _|_ {
```

Comment:
@binarylogic do you know of a better way to do this than the conditionals I ended up with?